### PR TITLE
scaffold-eth tutorial added

### PR DESCRIPTION
**Pull request by ayo-klaytn**
## Proposed changes
add scaffold-eth tutorial

## Types of changes

Please put an x in the boxes related to your change.

- [ ] Minor Issues and Typos
- [x] Major Content Contribution
- [ ] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to reach out. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [x] I have read the [CONTRIBUTING GUIDELINES](https://github.com/kaiachain/kaia-docs/blob/main/CONTRIBUTING.md)
- [x] I have read the [CLA](https://gist.github.com/kaiachain-dev/bbf65cc330275c057463c4c94ce787a6) and signed by comment ```I have read the CLA Document and I hereby sign the CLA``` in first time contribute
- [x] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Related issues

- Please leave the issue numbers or links related to this PR here.

## Further comments

If this is a relatively large content contribution, kick off the discussion by explaining why you would suggest the content contribution, etc...

### CODE REVIEW

File: docs/build/tutorials/scaffold-eth.md [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+
+Scaffold-ETH 2 is an open-source toolkit for building decentralized applications (dApps) on Ethereum and other EVM-compatible blockchains, like Kaia. Developers can easily deploy a Solidity smart contract and launch a dApp with a React frontend thanks to Scaffold-ETH 2.
+
+The Scaffold-ETH 2 toolkit was built using Next.js, RainbowKit, Hardhat, Foundry, Wagmi, and TypeScript. Developers can easily create, test, and deploy smart contracts using Hardhat or Foundry, as well as build a React frontend using Next.js.
+
+In this tutorial, you will learn how to deploy, run a contract and build a dApp on Kaia using Scaffold-ETH 2.
+
+## Prerequisites <a href="#prerequisites" id="prerequisites"></a>
+
+To get started in this guide, you will need:
```
Comment (scott-klaytn) [inline docs/build/tutorials/scaffold-eth.md:13]:
```suggestion
To get started with this guide, you will need:
```

File: docs/build/tutorials/scaffold-eth.md [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+kaia: {
+  url: "INSERT_RPC_URL",
+  accounts: [deployerPrivateKey],
+},
+```
+
+For more information on using Hardhat with Kaia, please check [Hardhat guide](https://docs.kaia.io/docs/build/get-started/hardhat/) for more details.
+
+### Deploy Contract to Kaia
+Having configured hardhat to support Kaia network, the next thing we want to do is to compile and deploy the sample contract.
```
Comment (scott-klaytn) [inline docs/build/tutorials/scaffold-eth.md:110]:
```suggestion
After configuring Hardhat to support the Kaia network, the next step is to compile and deploy the sample contract.
```

File: docs/build/tutorials/scaffold-eth.md [hunk 3]
```diff
@@ -0,0 +1,225 @@
...
+
+```js
+targetNetworks: [chains.klaytnBaobab],
+```
+
+That's all required to configure Next.js! Next, is to launch the dApp in your localhost. 
+
+### Launch the dApp in your Localhost <a href="#launch-dapp-in-localhost" id="launch-dapp-in-localhost"></a>
+
+Having made all necessary configurations, you can now launch the example dApp in your localhost.
```
Comment (scott-klaytn) [inline docs/build/tutorials/scaffold-eth.md:190]:
```suggestion
After making all the necessary configurations, you can now launch the example dApp on your localhost.
```

File: docs/build/tutorials/scaffold-eth.md [hunk 4]
```diff
@@ -0,0 +1,225 @@
...
+
+
+You should now be able to access a React-based dApp frontend at http://localhost:3000/. Feel free to  interact with the dApp by connecting your wallet or checking out the contract debugger page.
+
+![Scaffold dApp](/img/build/tutorials/sc-dapp.png)
+
+
+## Conclusion
+
+Congratulations on using the Scaffold-ETH 2 to deploy and run a contract and a dApp on Kaia. Now that you understand the workings of Scaffold-ETH 2, feel free to create and deploy your own smart contracts and modify the frontend to fit your dApp's needs!
```
Comment (scott-klaytn) [inline docs/build/tutorials/scaffold-eth.md:209]:
```suggestion
Congratulations! You have successfully used Scaffold-ETH 2 to deploy a contract and run a dApp on Kaia. Now that you understand the workings of Scaffold-ETH 2, feel free to create and deploy your own smart contracts and modify the frontend to fit your dApp's needs!
```

**Review by scott-klaytn — COMMENTED** [review]
Please check out a few simple suggestions for enhancement.

**Review by scott-klaytn — APPROVED** [review]

**Review by praveen-klaytn — APPROVED** [review]

**Pull request merged**